appenders/pythonAppender/setup.py

```diff
@@ -0,0 +1,9 @@
+from distutils.core import setup
+
+setup (
+    name='pythonAppender',
```

Comment:
Can we name this 'pulse_appender'? I can see this getting pushed to pypi at some point and in that case the 'python' part is redundant.